components/pivotgrid/overview.md

```diff
@@ -0,0 +1,243 @@
+
+
+## Configurator
+
+In the simpler case, the PivotGrid supports fixed configuraton of its rows, columns, and measures. However, most scenarios and users may require greater flexibility where one can [use the PivotGrid Configurator to change, filter and sort the displayed fields and aggregations]({%slug pivotgrid-configurator%}).
+
+
+## Templates
+
+[The PivotGrid supports templates for its header and data cells]({%slug pivotgrid-templates%}). The templates enable you to customize the cell content and appearance.
+
+
+## PivotGrid Parameters
+
+The tables below list the parameters of [all components, which comprise the PivotGrid](#components):
+
+* [Grid](#grid-parameters)
+    * [Rows, columns and measures](#row-column-and-measure-parameters)
+* [Configurator](#configurator-parameters)
+* [Button](#button-parameters)
+* [Container](#container-parameters)
+
+### Grid Parameters
+
+The following table lists the `TelerikPivotGrid` parameters. Also check the [PivotGrid API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikPivotGrid-1).
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | A custom CSS class for the `<div class="k-pivotgrid">` element. Use it to [override theme styles]({%slug themes-override%}). |
+| `Data` | `IEnumerable<TItem>` |  |
+| `DataProviderType` | `PivotGridDataProviderType` enum <br /> (`Local`) |  |
+| `EnableLoaderContainer` | `bool` (`true`) | Defines if a built-in [LoaderContainer]({%slug loadercontainer-overview%}) will show during long-running operations (over 600ms). |
+| `Height` | `string` | A `height` style in [any supported CSS unit]({%slug common-features/dimensions%}). |
+| `LoadOnDemand` | `bool` <br /> (`true`) | Defines if the PivotGrid will request only the data to display in the current view, or all data. When loading on demand is disabled or when using the `Local` `DataProviderType`, the component performs all calculations in-memory. In such cases, large amounts of data may impact the performance, especially in WebAssembly apps. |
+| `TItem` | `object` | The PivotGrid `@typeparam`. Required if the data item type cannot be inferred at compile-time. |
+| `Width` | `string` | A `width` style in [any supported CSS unit]({%slug common-features/dimensions%}). |
+
+### Row, Column and Measure Parameters
+
+The following table lists parameters of the `PivotGridRow`, `PivotGridColumn` and `PivotGridMeasure` tags.
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Aggregate` | `PivotGridAggregateType` enum <br /> (`Sum`) | The nature of the calculated aggregate value. Applies to `PivotGridMeasure` only. |
+| `Name` | `string` | The field name of the respective row, column or measure. |
+
+
+### Configurator Parameters
+
+The following table lists parameters of the `TelerikPivotGridConfigurator` component.
+
+| Parameter | Type | Description |
```

Comment:
The Configurator has its own `EnableLoaderContainer` parameter. Does it mean that it can be enabled/disabled separately? If so, we need to list it here as well.